picojson.h

```diff
@@ -76,8 +76,8 @@ extern "C" {
 // experimental support for int64_t (see README.mkdn for detail)
 #ifdef PICOJSON_USE_INT64
 #define __STDC_FORMAT_MACROS
-#include <errno.h>
-#include <inttypes.h>
+#include <cerrno>
+#include <cinttypes>
```

Comment:
Changing errno.h to cerrno seems correct.

OTOH, it is my understanding that cinttypes was introduced in C++11. Would you mind adding `#if` that includes the correct file based on the C++ version?

Reply:
Is TR1 low enough of a bar or do you want to also have `inttypes.h` included via `extern "C"`? 
Bare C++03 doesn’t seem to have support for `PRId64` on its own.